app/templates/components/event-card.hbs

```diff
@@ -9,25 +9,42 @@
       </div>
     {{/unless}}
   {{/if}}
-  <div style="height: 100%;"
-    class="ui card {{this.eventClass}}">
+  <div class="ui card {{this.eventClass}} {{if this.isWide 'h-auto' 'h-100'}}">
     {{#unless this.isWide}}
       <a class="image" href="{{href-to (if this.isCFS 'public.cfs' 'public') this.event.id}}">
         <Widgets::SafeImage
           @src={{if this.event.thumbnailImageUrl this.event.thumbnailImageUrl this.event.originalImageUrl}} />
       </a>
     {{/unless}}
-    <a class="main content" href="{{href-to (if this.isCFS 'public.cfs' 'public') this.event.id}}">
-      <SmartOverflow @class="header">
-        {{this.event.name}}
-      </SmartOverflow>
-      <div class="meta mb-2">
-        <span class="date">
-          <Public::EventDateTime @event={{@event}} @noBreak={{true}} />
-        </span>
-      </div>
-      <Public::EventVenue @event={{@event}} />
-    </a>
+
+    {{#if this.isWide}}
+      {{!-- On the search page --}} 
+            <a class="main content {{unless this.device.isMobile 'h-80perc'}}" href="{{href-to (if this.isCFS 'public.cfs' 'public') this.event.id}}">
+              <SmartOverflow @class="header">
+                {{this.event.name}}
+              </SmartOverflow>
+              <div class="meta mb-2">
+                <span class="date">
+                  <Public::EventDateTime @event={{@event}} @noBreak={{true}} />
+                </span>
+              </div>
+              <Public::EventVenue @event={{@event}} />
+            </a>
+      {{!-- On the home page --}}
+    {{else}}
+            <a class="main content" href="{{href-to (if this.isCFS 'public.cfs' 'public') this.event.id}}">
+              <SmartOverflow @class="header">
+                {{this.event.name}}
+              </SmartOverflow>
+              <div class="meta mb-2">
+                <span class="date">
+                  <Public::EventDateTime @event={{@event}} @noBreak={{true}} />
+                </span>
+              </div>
+              <Public::EventVenue @event={{@event}} />
+            </a>  
```

Comment:
duplication

Reply:
Yes ma'am, I agree there's repetition with very little change in the anchor tag code. What I meant to achieve was to make the height of the 'main content' anchor tag to 80% of the parent event card, so the hashtags would be pushed down. 
I am unable to implement a single inline conditional statement for the same.
I can implement a helper function to dealt with the conditions and reduce it to an inline function.
Would like to receive your suggestions on the same.

Reply:
Use helpers